app/src/main/java/com/example/therundown/domain/NbaAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.therundown.domain
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.therundown.R
+
+class NbaAdapter : RecyclerView.Adapter<NbaViewHolder>() {
+
+    private val dataSet = mutableListOf<Player>()
+
+    override fun onCreateViewHolder(viewGroup: ViewGroup, viewType: Int): NbaViewHolder {
+        val view = LayoutInflater.from(viewGroup.context)
+            .inflate(R.layout.nba_player, viewGroup, false)
+
+        return NbaViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: NbaViewHolder, position: Int) {
+        holder.bind(dataSet[position])
+    }
+
+    override fun getItemCount() = dataSet.size
+
+    fun add(newDataList: List<Player>){
```

Comment:
You can use ListAdapter instead of RecyclerViewAdapter (https://developer.android.com/reference/androidx/recyclerview/widget/ListAdapter)
or you can add TODO for implementing this in future.

Reply:
done